translation/templates.rst

```diff
@@ -0,0 +1,123 @@
+Using Translation in Templates
```

Comment:
Fyi: This article is simply a copy/paste from the main guide. It's far from perfect (explaining more a message formatter than a translation usage), but let's not focus on it in this PR